api-reference/upload-file-api/secure-client-side-file-upload.md

```diff
@@ -175,11 +86,9 @@ To create the JWT, the three outputs are transformed to Base64url strings and co
 The Private API key should be kept confidential and only stored on your own servers.
 {% endhint %}
 
-If you are using ImageKit.io [JavaScript SDK](https://github.com/imagekit-developer/imagekit-javascript) for file upload, it requires an `authenticationEndpoint` endpoint for getting authentication parameters required in the upload API.
-
-### How to implement authenticationEndpoint endpoint?
+### Backend token generation?
 
-This endpoint is specified by `authenticationEndpoint` parameter during initialization. The SDK sends an HTTP POST request to this endpoint and expects a JSON response having status code 200 containing the JWT in the `token` field.
+The client-side application sends an HTTP POST request to the backend and expects a JSON response having status code 200 containing the JWT in the `token` field.
```

Comment:
It is not necessary to be POST. And there is not expectation for a response structure. The developer is implementing it in a custom way, not constrained to SDK's parameter structure.

Reply:
done